codegen/protocol-test-codegen-local/model/eventstream-rpc.smithy

```diff
@@ -0,0 +1,83 @@
+namespace aws.protocoltests.eventstream
```

Comment:
This file is the same as `eventstream.smithy` except that the protocol is AWS JSON (i.e. a RPC protocol) instead of RestJSON.